.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: Build
+
+on:
+  push:
+    branches: [ release-1.15-intel ]
```

Comment:
You can just say `release-*-intel` so you need to update the file for every release.

Reply:
Fixed it